src/components/Base/Popover/index.scss

```diff
@@ -0,0 +1,115 @@
+@import '~scss/variables';
+
+.popover {
+  :global {
+    .pi-popover-popper {
+      padding: 7px 5px;
+      font-size: $body-size;
+      color: $title-color;
+      border: 1px solid $border;
+      background-color: #ffffff;
+      box-shadow: 0 2px 4px 0 rgba(48, 14, 86, 0.06), 0 1px 2px 0 rgba(48, 14, 86, 0.04);
+
+      .pi-popover-popper_arrow {
+        width: 16px;
+        height: 16px;
+  
+        &::before, &::after {
+          content: "";
+          position: absolute;
+          width: 0;
+          height: 0;
+          border-style: solid;
+          box-sizing: border-box;
+        }
+      }
+      
+      &[data-placement^="top"] {
+        margin-bottom: 8px;
+    
+        .pi-popover-popper_arrow {
+          bottom: -15px;
+    
+          &::before {
+            left: 0;
+            top: 2px;
+            border-width: 8px;
+            border-color: $border transparent transparent transparent;
+            z-index: 1;
+          }
+          &::after {
+            left: 1px;
+            top: 1px;
+            border-width: 7px;
+            border-color: #fff transparent transparent transparent;
+            z-index: 2;
+          }
+        }
+      }
+      &[data-placement^="bottom"] {
+        margin-top: 8px;
+    
+        .pi-popover-popper_arrow {
```

Comment:
`pi-popover-popper_arrow` seems repeated, can move it to a base section then extend it, or abstract it to a function

Reply:
The style here has the public part of the independence, not easily abstracted into a method.